x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/action/EqlSearchRequestTests.java

```diff
@@ -74,16 +83,53 @@ protected QueryBuilder parseQuery(XContentParser parser) throws IOException {
         return parseInnerQueryBuilder;
     }
 
-    private List<String> randomSearchAfter() {
-        if (randomBoolean()) {
-            return Collections.emptyList();
-        } else {
-            int size = randomIntBetween(1, 50);
-            List<String> arr = new ArrayList<>(size);
-            for (int i = 0; i < size; i++) {
-                arr.add(randomAlphaOfLength(randomIntBetween(1, 15)));
+    private SearchAfterBuilder randomJsonSearchFromBuilder() throws IOException {
+        int numSearchAfter = randomIntBetween(1, 10);
```

Comment:
How about an empty list?

Reply:
normally SearchAfter doesn't accept the empty list, throws error:
```
    java.lang.IllegalArgumentException: Values must contains at least one value.
```
The original searchafter test doesn't use empty list as well
https://github.com/elastic/elasticsearch/blob/df83eb983c5f5cdc50c3d2c3d080b4c56bc571f4/server/src/test/java/org/elasticsearch/search/searchafter/SearchAfterBuilderTests.java#L101

Reply:
How about returning setting search after values to null sometimes then?

Reply:
Already handled there as well
```
    java.lang.NullPointerException: Values cannot be null.
        at __randomizedtesting.SeedInfo.seed([1C63F70E75AEE77B:48BB964652FCDAE7]:0)
        at org.elasticsearch.search.searchafter.SearchAfterBuilder.setSortValues(SearchAfterBuilder.java:78)
```

Reply:
discussed over slack, updated implementation